### DISCUSSION TITLE
Welcome to Discord-MusicBot Discussions!

### DISCUSSION THREAD

**Discussion by ItzSudhan** [SudhanPlayz/Discord-MusicBot#68] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ItzSudhan** [comment]
Hello anyone here? lol

**Comment by Pancaananda** [comment]
Can I ask something ?

when I turn on the bot. why can't I trigger the Glitch Live site?
always starting in TAB
![image](https://user-images.githubusercontent.com/75909843/103927773-760fa300-514d-11eb-98df-64e3eb60d8af.png)

**Reply by ItzSudhan** [reply]
Because it doesnt have anything to show in the website for now, If you want then you can able to run a express server then
